library/common/strings/string_conversions.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+#include <string>
+
```

Comment:
I think these methods should probably be in a namespace, right?

Reply:
Done.